desktop/menus/view-menu.js

```diff
@@ -5,6 +5,32 @@ const buildViewMenu = (settings, isAuthenticated) => {
   settings = settings || {};
   isAuthenticated = isAuthenticated || false;
 
+  const themeSubMenu = platform.isLinux()
+    ? [
+        {
+          label: '&Light',
+          id: 'light',
+        },
+        {
+          label: '&Dark',
+          id: 'dark',
+        },
+      ]
+    : [
+        {
+          label: '&System',
+          id: 'system',
+        },
+        {
+          label: '&Light',
+          id: 'light',
+        },
+        {
+          label: '&Dark',
+          id: 'dark',
+        },
+      ];
```

Comment:
I think that below looks cleaner

```
const themeSubMenu = [
  ...(platform.isLinux() ? [] : [{ label: "&System", id: "system" }]),
  { label: "&Light", id: "light" },
  { label: "&Dark", id: "dark" },
];

```

Reply:
Thanks. I can see how that is not the most readable, I've updated it but using a third option that to me is the most readable.